### PR TITLE
Update contractkit wrappers to match release 1 contracts.

### PR DESCRIPTION
**Pull request by ewilz**
### Description
** Will Merge when `Contracts Release 1` CGP is officially passed **
Update contractkit wrappers to match release 1 contracts. Governance wrapper was already caught up. I excluded one attestations new function because it's only valuable to call it on-chain since it is merely a require statement. If reviewing please double check the release report to make sure I didn't miss any function changes in this PR.

Changes include:
- [x] New `Exchange` functions: (`sell()`/`buy()` instead of `exchange()`)
- [x] `Attestations` transferring: (`approveTransfer()` )
- [x] `Governance getVoteRecord` new signature (already added!)

### Other changes

added expectations to tests without expectations. 

### Tested

Did not test attestations function due to lack of sufficient test setup. I can do this if it's something we'd like to improve this sprint though.

### Related issues

- https://github.com/celo-org/celo-monorepo/issues/5408

### Backwards compatibility

yes

### CODE REVIEW

**Comment by emaG3m** [comment]
Not ready for review, adding one more big update.

File: packages/contractkit/src/wrappers/Attestations.ts [hunk 1]
```diff
@@ -456,6 +456,28 @@ export class AttestationsWrapper extends BaseWrapper<Attestations> {
...
+   * @param status The approval status
+   */
+  async approveTransfer(
+    identifier: string,
+    index: number,
+    from: Address,
+    to: Address,
+    status: boolean
+  ) {
+    return toTransactionObject(
```
Comment (ewilz) [inline packages/contractkit/src/wrappers/Attestations.ts]: Is it possible we'd like to throw errors in this kit if we can predict an on-chain revert? Seems potentially useful, but I need to know how to access the address of `msg.sender` to do that here. Is that possible?

Comment (barbaraliau) [reply packages/contractkit/src/wrappers/Attestations.ts]: Hm. I don't know if you can access `msg.sender` in the wrappers.

**Comment by barbaraliau** [comment]
@ewilz Can you link the release report?

**Comment by ewilz** [comment]
@barbaraliau  https://github.com/celo-org/celo-monorepo/releases/tag/celo-contracts-v1.rc1

File: packages/contractkit/src/wrappers/Exchange.ts [hunk 2]
```diff
@@ -61,6 +61,64 @@ export class ExchangeWrapper extends BaseWrapper<Exchange> {
...
+   * DEPRECATED: use function sell
+   * Exchanges sellAmount of sellToken in exchange for at least minBuyAmount of buyToken
+   * Requires the sellAmount to have been approved to the exchange
+   * @param sellAmount The amount of sellToken the user is selling to the exchange
+   * @param minBuyAmount The minimum amount of buyToken the user has to receive for this
+   * transaction to succeed
+   * @param sellGold `true` if gold is the sell token
+   * @return The amount of buyToken that was transfered
+   */
+  exchange: (
```
Comment (ewilz) [inline packages/contractkit/src/wrappers/Exchange.ts:74]: just relocated this function and put DEPRECATED in doc.

File: packages/contractkit/src/wrappers/Attestations.ts (hunk 1 above)
Comment (nambrot) [reply packages/contractkit/src/wrappers/Attestations.ts]: I think it's ok to not do these kind of runtime checks. At the latest, gas estimation will show that

Comment (nambrot) [reply packages/contractkit/src/wrappers/Attestations.ts]: If you are just "exposing" the method on the wrapper, you can use the higher order function `proxySend` here as well

**Review by nambrot — APPROVED** [review]
Just one minor comment to use proxySend

**Pull request merged**